Disable pretty printers so that they do not fail the build until
rhbz#1403329 is fixed.

Index: b/nptl/nptl_lock_constants.pysym
===================================================================
--- a/nptl/nptl_lock_constants.pysym
+++ b/nptl/nptl_lock_constants.pysym
@@ -51,11 +51,6 @@ PTHREAD_COND_SHARED             (void *)
 -- Value of __total_seq for destroyed condvars.
 PTHREAD_COND_DESTROYED          -1ull
 
--- __nwaiters encodes the number of threads waiting on a condvar
--- and the clock ID.
--- __nwaiters >> COND_NWAITERS_SHIFT gives us the number of waiters.
-COND_NWAITERS_SHIFT
-
 -- Condvar clock IDs
 CLOCK_REALTIME
 CLOCK_MONOTONIC
